examples/bearriver/src/FRP/BearRiver.hs

```diff
@@ -205,7 +220,7 @@ b0 --> sf = sf >>> replaceOnce b0
 a0 >-- sf = replaceOnce a0 >>> sf
 
 replaceOnce :: Monad m => a -> SF m a a
-replaceOnce a = dSwitch (arr $ const (a, Event ())) (const $ arr id)
+replaceOnce a = dSwitch (constant (a, Event ())) (const $ arr id)
```

Comment:
You can refactor `const $ arr id` as well, I think. It should even work with `Category.id`.

Reply:
Just a note: It seems like this change is unrelated to your pull request.

Reply:
Good point. And, yes, it helps to keep PR straight to the point and focus on one feature only. It makes the maintainer live's so much easier.

Reply:
Yes you got me there, I thought I could sneak it in as general "refactoring". Will do it next time properly, sorry.